drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -594,7 +594,7 @@ static uint8_t nrf5_get_acc(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
-	return CONFIG_IEEE802154_DELAY_TRX_ACC;
+	return CONFIG_IEEE802154_NRF5_DELAY_TRX_ACC;
```

Comment:
This produces warning and breaks CI, since it can be larger than UINT8_MAX.
https://github.com/zephyrproject-rtos/zephyr/pull/44402/checks?check_run_id=5769581818